ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +73,97 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def get_drpc_name(namespace):
+    """
+    Get the DRPC Resource Name
+    Args:
```

Comment:
Missing new lines in-between.
Same for other functions.

Reply:
Done